datafusion/physical-expr/src/expressions/in_list.rs

```diff
@@ -32,13 +33,15 @@ use arrow::{
     record_batch::RecordBatch,
 };
 
-use crate::PhysicalExpr;
+use crate::{expressions, PhysicalExpr};
 use arrow::array::*;
 use arrow::buffer::{Buffer, MutableBuffer};
 use datafusion_common::ScalarValue;
 use datafusion_common::{DataFusionError, Result};
 use datafusion_expr::ColumnarValue;
 
+static OPTIMIZER_INSET_THRESHOLD: usize = 400;
```

Comment:
```suggestion
/// Size at which to use a Set rather than Vec for `IN` / `NOT IN`
/// Value chosen to be consistent with Spark
static OPTIMIZER_INSET_THRESHOLD: usize = 400;
```

Reply:
@Ted-Jiang  do you  happen to  have a link we could point to (as in I don't know how to verify this claim, I am just copying it from the PR comments)?